### PR TITLE
feat(admin-cli): support nodes capacity balance using admin-cli

### PR DESCRIPTION
**Pull request by foreverneverer**
# Related-Issue
https://github.com/apache/incubator-pegasus/issues/962

# Problem
By default, the node capacity of the server needs to be updated every 10 minutes. Therefore, after a partition is migrated completed, the tool cannot immediately obtain the latest capacity distribution. Please adjust the node capacity update interval of the server to speed up the equalization speed. Relevant configurations are as follows:
```diff
- disk_stat_interval_seconds = 600
+ - disk_stat_interval_seconds = 60 # or less

- gc_memory_replica_interval_ms = 600000
+ gc_memory_replica_interval_ms = 60000 # or less
```

### CODE REVIEW

**Review by hycdong — DISMISSED** [review]

File: admin-cli/executor/toolkits/nodesbalancer/migrator.go [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+ */
+
+package nodesbalancer
+
+import (
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/XiaoMi/pegasus-go-client/idl/base"
```
Comment (acelyc111) [inline admin-cli/executor/toolkits/nodesbalancer/migrator.go:27]: need update

File: admin-cli/executor/toolkits/nodesbalancer/balancer.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	if err != nil {
+		return err
+	}
+	toolkits.LogInfo("set meta.lb.only_primary_balancer true")
+	err = executor.RemoteCommand(client, session.NodeTypeMeta, "", "meta.lb.only_primary_balancer", []string{"true"})
+	if err != nil {
+		return err
+	}
+	// reset garbage replica clear interval
+	toolkits.LogInfo("set gc_disk_error_replica_interval_seconds 10")
```
Comment (acelyc111) [inline admin-cli/executor/toolkits/nodesbalancer/balancer.go:100]: Should we restore these configs after this rebalance operation?

File: admin-cli/executor/toolkits/diskbalancer/migrator.go [hunk 3]
```diff
@@ -68,7 +68,7 @@ func changeDiskCleanerInterval(client *executor.Client, replicaServer string, cl
 }
 
 func getNextMigrateAction(client *executor.Client, replicaServer string, minSize int64) (*MigrateAction, error) {
-	disks, totalUsage, totalCapacity, err := queryDiskCapacityInfo(client, replicaServer)
+	disks, totalUsage, totalCapacity, err := QueryDiskCapacityInfo(client, replicaServer)
```
[the remark this answers is not in the document]
Comment (foreverneverer) [reply admin-cli/executor/toolkits/diskbalancer/migrator.go:71]: uppercase is means `public` in go, which can be used for other file

File: admin-cli/executor/toolkits/nodesbalancer/balancer.go [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+import (
+	"fmt"
+	"time"
+
+	"github.com/XiaoMi/pegasus-go-client/session"
+	"github.com/apache/incubator-pegasus/admin-cli/executor"
+	"github.com/apache/incubator-pegasus/admin-cli/executor/toolkits"
+)
+
+// By default, the node capacity of the server needs to be updated every 10 minutes.
```
[the remark this answers is not in the document]
Comment (foreverneverer) [reply admin-cli/executor/toolkits/nodesbalancer/balancer.go:31]: The config hasn't be supported query, so we just add hint for user

File: admin-cli/executor/toolkits/nodesbalancer/balancer.go (hunk 2 above)
Comment (foreverneverer) [reply admin-cli/executor/toolkits/nodesbalancer/balancer.go:100]: Yeah, I will recover it

**Review by acelyc111 — APPROVED** [review]

**Review by hycdong — APPROVED** [review]

**Pull request merged**